SCons/Tool/msvc.xml

```diff
@@ -412,6 +413,14 @@ you don't want &SCons; to change anything.
 </summary>
 </cvar>
 
+<cvar name="MSVC_USE_SCRIPT_ARGS">
+<summary>
+<para>
+Provides arguments passed to the script &cv-MSVC_USE_SCRIPT;.
```

Comment:
Nitpick: since this is a reference to a different consvar, prefer using linked form: `&cv-link-MSVC_USE_SCRIPT;`.  Perhaps the maintainer can just edit this change in?